### PR TITLE
fix(packagers): check real outdir for pnpm

### PR DESCRIPTION
**Pull request by millsp**
Test will fail until https://github.com/prisma/prisma/pull/12427 is merged.

### CODE REVIEW

File: packagers/pnpm-workspaces-default-output/test.sh [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-FILE0=node_modules/.prisma/client/index.js
 FILE1=sub-project-1/node_modules/.prisma/client/index.js
 FILE2=sub-project-2/node_modules/.prisma/client/index.js
 
 set -eux
 
-cd workspace
+cd workspace/sub-project-1
+FILE0=$(node -e "console.log(path.join(require.resolve('@prisma/client/package.json'), '..', '..', '..', '.prisma'))")
```
Comment (Jolg42) [inline packagers/pnpm-workspaces-default-output/test.sh:9]: What's the resolved path of this?

Comment (millsp) [reply packagers/pnpm-workspaces-default-output/test.sh:9]: Something like
`/home/millsp/Work/e2e-tests/packagers/pnpm-workspaces-default-output/workspace/node_modules/.pnpm/@prisma+client@3.12.0-integration-fix-yarn1-workspaces.2_663dcaeee949835cfab1a843c306e085/node_modules/.prisma`

Comment (Jolg42) [inline packagers/pnpm-workspaces-default-output/test.sh:9]: Could be nice to add a comment about what this is doing

**Review by Jolg42 — APPROVED** [review]

**Review by aqrln — APPROVED** [review]

File: packagers/pnpm-workspaces-default-output/test.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ if [ -f "$FILE1" ] || [ -f "$FILE2" ]; then
     echo "Client should not generate in sub-folder"; exit 1;
 fi;
 
-if [ ! -f "$FILE0" ]; then
+if [ ! -d "$FILE0" ]; then
```
Comment (Jolg42) [inline packagers/pnpm-workspaces-default-output/test.sh:15]: nitpick:
-f is for checking if the file exist and -d for a directory Ithink @millsp?
So it's not a file like `$FILE0` suggests

**Pull request merged**